account_banking_pain_base/models/account_payment_order.py

```diff
@@ -497,7 +497,7 @@ def generate_party_block(
             party_type_label = _("Creditor name")
         elif party_type == "Dbtr":
             party_type_label = _("Debtor name")
-        name = "partner_bank.partner_id.name"
+        name = 'partner_bank.acc_holder_name or partner_bank.partner_id.name'
```

Comment:
This will fail pre-commit, as it uses black with double quote instead of simple.

Reply:
fixed.